myndla/src/main/scala/no/ndla/myndla/repository/UserRepository.scala

```diff
@@ -24,25 +25,41 @@ trait UserRepository {
 
   class UserRepository extends StrictLogging {
 
-    def getUsersPaginated(offset: Long, limit: Long)(implicit session: DBSession) = Try {
+    def getUsersPaginated(offset: Long, limit: Long, filterTeachers: Boolean, query: Option[String])(implicit
+        session: DBSession
+    ): Try[(Long, List[MyNDLAUser])] = Try {
       val u = DBMyNDLAUser.syntax("u")
-      sql"""
+
+      val teacherClause = Option.when(filterTeachers)(sqls"u.document->>'userRole' = ${UserRole.EMPLOYEE.toString}")
+      val queryClause = query.map(q => {
+        val qString = s"%$q%"
+        sqls"u.document->>'displayName' ilike $qString or u.document->>'username' ilike $qString"
```

Comment:
Eller er ilike noko eg ikkje veit om?
```suggestion
        sqls"u.document->>'displayName' like $qString or u.document->>'username' ilike $qString"
```

Reply:
`ilike` er det samme som `like` bare case-insensitive

Reply:
Ooo. Der lærte eg nokke nytt!